python/lsst/analysis/tools/atools/metadataMetrics.py

```diff
@@ -0,0 +1,54 @@
+# This file is part of analysis_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+__all__ = ("MetadataMetricTool",)
+
+from typing import Any, Dict, Iterable
+
+from lsst.pex.config import DictField, Field
+
+from ..interfaces import AnalysisAction, AnalysisTool
+
+
+class LoadDoubleKeyedData(AnalysisAction):
+    """Load double keyed data"""
```

Comment:
Can you be more explicit here?

Reply:
Furthermore, this probably needs to be moved to somewhere under the `action` namespace.

Reply:
I've made the docstring more specific. The input data is something like:
```python
{'AMPOFFSET_PEDESTALS': {'0': -0.20423207279935987, '1': -0.06727251761094916, '2': -0.28773895061785293, '3': 0.5592435410281624}}
```

Reply:
Not sure where would be the best place to move it to, maybe above [this class](https://github.com/lsst/analysis_tools/blob/cdf17bb5f2d4739b48d7302d31eab345eaa8c960/python/lsst/analysis/tools/actions/vector/vectorActions.py#L71)?

Reply:
[I'd imagine somewhere in here would be optimal](https://github.com/lsst/analysis_tools/blob/main/python/lsst/analysis/tools/actions/keyedData/keyedDataActions.py)? You can probably change it to inherit from `KeyedDataAction` instead as well (please test).

Reply:
I was unsure about `KeyedDataAction` at first because I thought it had to change the data, not just load it. On second thought, I think it might be the right place. Testing it now.

Reply:
Made a more generic action named `KeyedDataKeyAccessAction` that inherits from `KeyedDataAction` and moved it to `keyedDataActions.py`.